src/RelayServer.ts

```diff
@@ -495,26 +500,42 @@ export class RelayServer extends EventEmitter {
     async validateViewCallSucceeds(
```

Comment:
This name can be improved. The main purpose of the function is to find the amount of gas required for the function to succeed. I'm thinking on findGasLimit, getGasLimit, findGasLimitForSuccess, findMinimumGas. For sure you can find a better name. You're the naming king of the team.

Reply:
updated